VERSIONS.txt

```diff
@@ -6,9 +6,9 @@ harfbuzz                                        release     2.6.1
 skia                                            release     m68
 xunit                                           release     2.4.1
 xunit.runner.console                            release     2.4.1
-Xamarin.Forms                                   release     4.2.0.910310
-Xamarin.Forms.Platform.WPF                      release     4.2.0.910310
-Xamarin.Forms.Platform.GTK                      release     4.2.0.910310
+Xamarin.Forms                                   release     4.6.0.280-nightly
```

Comment:
Do we need version 4.6. will it work with version 4.5?

Reply:
Oh, I am testing things. The build keeps failing for UWP. I am working with the Forms team to figure this out. But, yes, it will be 4.5 when it goes out.